messages/en.json

```diff
@@ -177,6 +177,8 @@
     "BUY": "Buy",
     "NAVIGATE": "Navigate",
     "SEARCH_CENTER": "Set Search Center",
+    "SELL_CENTER": "Set Sell Center",
```

Comment:
This translation already exists in the `SCREEN.SELLER_REGISTRATION.SELLER_SELL_CENTER`.